main.cpp

```diff
@@ -323,30 +406,36 @@ Thing 5) wheels
  */
 struct Wheels
 {
-
-    //wheel diameter
     int wheelDiameter = 50;
-    //wheel width
     float wheelWidth = 27.4f;
-    //amount of lug nuts
     int amountLugNuts = 30;
-    //tyre thread depth
     double tyreThreadDepth = 20.2;
-    //tyre pressure
     float tyrePressure = 15.3f;
 
-    //carry vehicle
-    void carryVehicle();
-    //aid suspension
-    void aidSuspension(float maxPressureSetting);
-    //turn vehicle
+    bool carryVehicle(int numberOfWheels);
+    bool aidSuspension(float maxPressureSetting);
     void turnVehicle(double SetmaxDegree);
 
 };
 
+bool Wheels::carryVehicle(int numberOfWheels)
+{
+  if(numberOfWheels >=4)
+    return true;
+    return false;
+}
 
+bool Wheels::aidSuspension(float maxPressureSetting)
+{
+    if(tyrePressure - maxPressureSetting <=50)
+    return true;
+    return false;
+}
```

Comment:
Fix this according to my previous pull request review instructions
- [ ] revise